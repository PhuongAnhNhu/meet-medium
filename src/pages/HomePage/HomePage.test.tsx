import { render } from '@testing-library/react';
import { wrapWithStore } from 'tests/wrappers';
import Homepage from '.';

describe('HomePage', () => {
  it('renders', () => {
    const { container } = render(wrapWithStore(<Homepage />));

    expect(container).toMatchSnapshot();
  });
});
