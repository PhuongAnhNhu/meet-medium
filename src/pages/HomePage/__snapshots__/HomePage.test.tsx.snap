// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage renders 1`] = `
<div>
  <div
    class="MuiBox-root css-1lmhv1o"
  >
    <div
      class="MuiGrid-root MuiGrid-container css-lt4bf8-MuiGrid-root"
    />
  </div>
</div>
`;
